tools/IoTEdgeDevOps/DevOpsLib/BuildExtension.cs

```diff
@@ -16,15 +16,14 @@ public static class BuildExtension
                 BuildDefinitionId.EndToEndTest,
                 BuildDefinitionId.NestedEndToEndTest,
                 BuildDefinitionId.ConnectivityTest,
-                BuildDefinitionId.NestedConnectivityTest,
                 BuildDefinitionId.LonghaulTestEnv1,
                 BuildDefinitionId.LonghaulTestEnv2,
                 BuildDefinitionId.LonghaulTestEnv3,
                 BuildDefinitionId.NestedLonghaulTest,
                 BuildDefinitionId.StressTestEnv1,
                 BuildDefinitionId.StressTestEnv2,
                 BuildDefinitionId.StressTestEnv3,
-                BuildDefinitionId.NestedISA95
+                BuildDefinitionId.NestedISA95,
```

Comment:
typo : `,`

Reply:
There is another trailing comma right here below already checked in:
https://github.com/Azure/iotedge/pull/5646/files#diff-83be14ea102e07dec10537d395a5633ff97ec20b1ecc76fe93ae18e8b8ba72d9R47

I will still remove both as discussed. I think we aren't consistent with this though because we have it in some places but not others.